pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -226,13 +226,13 @@ func (s *ServiceImpl) getHomeNode(c *contextmodel.ReqContext, prefs *pref.Prefer
 		SortWeight: navtree.WeightHome,
 	}
 	ctx := c.Req.Context()
-	if s.features.IsEnabled(ctx, featuremgmt.FlagHomeSetupGuide) {
+	if _, exists := s.pluginStore.Plugin(ctx, "grafana-setupguide-app"); exists {
```

Comment:
should we remove the `featuremgmt.FlagHomeSetupGuide` then? I think it's no more used

Reply:
This PR does remove it?

Reply:
sorry, my mistake: it does!